deploy/FoxStaking.ts

```diff
@@ -18,7 +17,7 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
 
   await deploy("FoxStaking", {
     from: admin,
-    args: [fox.address, foxy.address, epochLength, firstEpochNumber, firstEpochBlock ],
+    args: [foxy.address, epochLength, firstEpochNumber, firstEpochBlock ],
```

Comment:
I would not hardcode the fox address.  Plan is that in the future other DAOs may want to use this same set of contracts so allowing the token address to be easily passed in should help facilitate this and the gas savings is trivial with immutable vs constant

Reply:
If there is time, I would love to try the same proxy contract pattern as TOKE.  I think having this in mind it would be a good way to do it